src/vfxt/src/createUiDefinition.json

```diff
@@ -151,10 +151,10 @@
               "name": "adminPassword",
               "type": "Microsoft.Common.PasswordBox",
               "label": {
-                "password": "vFXT Node Password",
+                "password": "Cluster administration password",
```

Comment:
Is "admin" better than "administration" in this case? I could argue either side.